.github/workflows/phpunit-sqlite.yml

```diff
@@ -42,20 +61,15 @@ jobs:
   phpunit-sqlite:
     runs-on: ubuntu-latest
 
-    needs: changes
+    needs: [changes, matrix]
     if: needs.changes.outputs.src != 'false'
 
     strategy:
       matrix:
-        php-versions: ['8.1', '8.2', '8.3']
-        server-versions: ['master']
-        include:
-          - php-versions: '8.3'
-            server-versions: 'stable29'
-          - php-versions: '8.3'
-            server-versions: 'stable28'
-          - php-versions: '8.0'
-            server-versions: 'stable27'
+        php-versions: ${{ fromJson(needs.matrix.outputs.php-version) }}
+        server-versions: ${{ fromJson(needs.matrix.outputs.server-max) }}
```

Comment:
This means phpunit will only run on latest server release and no longer on `stable28` and `stable29`, right? I'm fine with this change, just wanted to make sure it's on purpose.

Reply:
that's right. mysql would test the full matrix it seems. Let me add it